workalendar/core.py

```diff
@@ -327,6 +327,19 @@ def get_whit_sunday(self, year):
     def get_corpus_christi(self, year):
         return self.get_easter_sunday(year) + timedelta(days=60)
 
+    def shift_christmas_boxing_days(self, year):
```

Comment:
I think you can add a docstring here, for more clarity. it's not a blocking point, though